docs/user_guide/plugin_development/messaging.rst

```diff
@@ -110,12 +106,12 @@ like so:
         @botcmd(template="hello")
         def hello(self, msg, args):
             """Say hello to someone"""
-            response = tenv().get_template('hello.html').render(name=args)
+            response = tenv().get_template('hello.md').render(name=args)
             self.send(msg.frm, response, message_type=msg.type)
 
 .. note::
     In both cases a plain-text version of your template is automatically
```

Comment:
yeah, this is obsolete: before we did an horrible hack job doing im-xtml to plain text conversion.

Now it is markdown to whatever the backend supports.

So you can simply remove the note IMHO.